sway-lib-std/src/primitive_conversions/b256.sw

```diff
@@ -6,6 +6,10 @@ use ::option::Option::{self, *};
 use ::u128::U128;
 use ::b512::B512;
 
+// TODO: Delete this implementation in favor of the one in `std::bytes`.
+//       Due the fact that we still don't have trait coherence, these
+//       two impls coexist for now.
+#[cfg(experimental_try_from_bytes_for_b256 = false)]
```

Comment:
The implementations are semantically equivalent AFAICT. It's a shame we can't solve this by replacing one of the implementations with a `pub use` without also exporting a lot of other stuff.